option/internaloption/internaloption.go

```diff
@@ -50,3 +50,18 @@ type skipDialSettingsValidation struct{}
 func (s skipDialSettingsValidation) Apply(settings *internal.DialSettings) {
 	settings.SkipValidation = true
 }
+
+// EnableDirectPath returns a ClientOption that overrides the default
+// attempt to use DirectPath.
+//
+// It should only be used by Yoshi libraries.
```

Comment:
nit: "It should only be used internally by generated clients."

This package is better labeled experimental, so let's just follow the convention from the functions and state something like this.

Reply:
Done.